.ort.yml

```diff
@@ -1,11 +1,11 @@
 excludes:
   paths:
-  - pattern: "analyzer/src/funTest/assets/projects/**"
+  - pattern: "{analyzer,scanner,reporter}/src/{funTest,test}/**"
     reason : "TEST_TOOL_OF"
-    comment: "This project is only used for testing and is not part of the distribution."
-  - pattern: "reporter/src/{funTest,test}/assets/**"
+    comment: "Licenses contained in this directory are used for testing and do not apply to the OSS Review Toolkit."
+  - pattern: "{downloader,evaluator,model,utils}/src/test/**"
     reason : "TEST_TOOL_OF"
-    comment: "This file contains test data. Contained licenses do not apply to the OSS Review Toolkit."
+    comment: "Licenses contained in this directory are used for testing and do not apply to the OSS Review Toolkit." 
```

Comment:
Some submodules are still missing, e.g. `cli`. To simplify this we should replace above two excludes with pattern `*/src/{funTest,test}/**`. The entry for `spdx-utils` below could also be removed then.

Reply:
Good idea, done.